hack/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM golang:1.7.3
+RUN curl https://glide.sh/get | sh
```

Comment:
> this isn't root on your machine, its just root in the container - so its safe

Docker containers share the host machine's kernel. Do not overestimate how safe it is to run things as root in a container.

So, I agree with @pmorie's point. Here's a safer way to install a _specific_ version of glide (which is important for the sake of determinism).

I've cribbed this from the [Dockerfile for Deis' containerized development environment](https://github.com/deis/docker-go-dev/blob/master/rootfs/Dockerfile):

```
# ...

ENV GLIDE_VERSION=v0.12.3

# ...

RUN curl -sSL https://github.com/Masterminds/glide/releases/download/$GLIDE_VERSION/glide-$GLIDE_VERSION-linux-amd64.tar.gz \
    | tar -vxz -C /usr/local/bin --strip=1
```